Tests/Foundation/Tests/TestFileManager.swift

```diff
@@ -934,6 +934,84 @@ class TestFileManager : XCTestCase {
         }
         XCTAssertNil(try? fm.linkItem(atPath: srcLink, toPath: destLink), "Creating link where one already exists")
     }
+    
+    func test_resolvingSymlinksInPath() throws {
```

Comment:
I take it this test would fail without this specific change?

Reply:
Exactly.